Sources/Plasma/Apps/plPythonPack/main.cpp

```diff
@@ -426,61 +342,69 @@ void PackDirectory(const plFileName& dir, const plFileName& rootPath, const plFi
     }
 
     s.Close();
+    ST::printf(out, "\nPython Package written to {}\n", pakName);
 
     PythonInterface::finiPython();
 }
 
 void PrintUsage()
 {
-    fputs("Usage:\n", stdout);
-    fputs("plPythonPack [directory to pack...]\n", stdout);
-    fputs("NOTE: the directory to pack must have full system and plasma dirs and\n", stdout);
-    fputs("      must be a relative path to the current working directory\n", stdout);
+    ST::printf(out, "The Python Pack Utility\n");
+    ST::printf(out, "Usage:\n");
+    ST::printf(out, "plPythonPack [options] <directory to pack...>\n");
+    ST::printf(out, "NOTE: the directory to pack must have full system and plasma dirs.\n");
+    ST::printf(out, "\nAvailable options:\n");
+    ST::printf(out, "\t-q\tQuiet  - Only print errors\n");
+    ST::printf(out, "\t-h\tHelp   - Print this help\n");
 }
 
 int main(int argc, char *argv[])
 {
-    fputs("The Python Pack Utility\n", stdout);
-
-    plFileName baseWorkingDir = plFileSystem::GetCWD();
+    // Parse arguments
+    ST::string packDir = ".";
+
+    enum { kArgPath, kArgQuiet, kArgHelp1, kArgHelp2 };
+    const plCmdArgDef cmdLineArgs[] = {
+        { kCmdArgOptional | kCmdTypeString, "path",  kArgPath},
+        { kCmdArgFlagged  | kCmdTypeBool,   "quiet", kArgQuiet},
+        { kCmdArgFlagged  | kCmdTypeBool,   "help",  kArgHelp1},
+        { kCmdArgFlagged  | kCmdTypeBool,   "?",     kArgHelp2},
+    };
+
+    std::vector<ST::string> args;
+    args.reserve(argc);
+    for (size_t i = 0; i < argc; i++) {
+        args.push_back(ST::string::from_utf8(argv[i]));
```

Comment:
`emplace_back` to avoid copying a temporary

Reply:
:trollface: 
https://github.com/H-uru/Plasma/blob/master/Sources/Plasma/Apps/plUruLauncher/plClientLauncher.cpp#L458-L462